packages/bundle-source/package.json

```diff
@@ -22,9 +22,9 @@
     "ava": "^3.12.1"
   },
   "dependencies": {
-    "@babel/generator": "^7.14.2",
-    "@babel/parser": "^7.14.2",
-    "@babel/traverse": "^7.14.2",
+    "@agoric/babel-generator": "^7.17.3",
```

Comment:
Have we considered using a resolutions instead / in addition?

Reply:
Resolutions are useful, but unless I’m mistaken, don’t get inherited up to the application. I am hoping that this change makes duplication less of an existential problem too!

Reply:
Yeah it's unfortunately hard to tell if a dependency is dev, prod or both in `yarn.lock` alone.